src/config/sites_fixes_v2.json

```diff
@@ -44,6 +44,10 @@
             ]
         },
         {
+            "url": "facebook.github.io/react/*",
+            "selectors":".hero, .nav-main, .nav-home"
```

Comment:
Seems I should write a Wiki for editing this config and provide some panel for testing selectors to prevent unexpected results.
You should be attentive. There is a `{common}` keyword, which inserts Common Selectors (img, iframe etc.). If it will not be specified, the images and other content specified in `commonSelectors` will become inverted.